style.css

```diff
@@ -0,0 +1,449 @@
+  text-align: center;
+  letter-spacing: 0.2px;
+  color: #737373;
+  max-width: 470px;
+}
+.section-card__title {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0.1px;
+  color: #252b42;
+}
+.section-card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 35px 40px;
+  gap: 15px;
+  width: 328px;
+  height: 193px;
+  background-color: #ffffff;
+  border: 1px solid #e8e8e8;
+  box-sizing: border-box;
+}
+.section-card__icon {
+  width: 47.32px;
+  height: 40.56px;
+}
+.section-card__header {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+.section__description--card {
+  max-width: 245px;
+  text-align: left;
+}
+.markerting-section__cards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px;
+  gap: 30px;
+  width: 1046px;
+  height: 416px;
+}
+.markerting-section__cards-row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 30px;
+  width: 1046px;
+  height: 193px;
+}
+.service-section {
+  background-color: #fafafa;
+  gap: 50px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  padding: 80px 0px;
+  position: relative;
+}
+.service-section__content {
+  position: relative;
+  z-index: 10;
+}
+.service-section__img {
+  width: 877px;
+  height: 442px;
+
+  background: linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0) 14.58%,
+    rgba(56, 56, 56, 0.84) 100%
+  );
+}
+
+
+.client-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 30px 35px;
+  width: 257px;
+  /* height: 244px; */
+  background-color: #ffffff;
+  border: 1px solid #dedede;
+  border-radius: 5px;
+  gap: 20px;
+}
+.client-card__icon {
+  width: 110.07px;
+  height: 22px;
+}
+.client-card__description {
+  font-size: 14px;
+  line-height: 24px;
+  letter-spacing: 0.2px;
+  color: #737373;
+  max-width: 234px;
+}
+.client-card__footer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: 15px;
+  width: 160px;
+  height: 50px;
+}
+.client-card__footer-names {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  width: 95px;
+  height: 48px;
+  gap: 5px;
+}
+.client-card__footer-name {
+  font-weight: 600;
+  font-size: 14px; 
+  line-height: 24px;
+  max-width: 95px;
+  text-decoration: none;
+  color: #ffa62b;
+}
+.general-section__cards {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: 30px;
+  width: 1048px;
+  height: 244px;
+}
+
+.general-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px 0px;
+  gap: 80px;
+}
+.team-card {
+  box-sizing: border;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px;
+  width: 328px;
+  height: 373px;
+  background-color: #ffffff;
+  border: 1px solid #eeeeee;
+}
+.general-section__cards--big-size{
```

Comment:
this modifier should be up with general section not in the middle of the style of the other block